cmd/gaia/app/genesis.go

```diff
@@ -405,13 +393,9 @@ func CollectStdTxs(cdc *codec.Codec, moniker string, genTxsDir string, genDoc tm
 
 func NewDefaultGenesisAccount(addr sdk.AccAddress) GenesisAccount {
 	accAuth := auth.NewBaseAccountWithAddress(addr)
-	coins := sdk.Coins{
+	accAuth.Coins = sdk.NewCoins(
 		sdk.NewCoin("footoken", sdk.NewInt(1000)),
 		sdk.NewCoin(defaultBondDenom, freeTokensPerAcc),
-	}
-
-	coins.Sort()
-
-	accAuth.Coins = coins
+	)
```

Comment:
```suggestion
	).Sort()
```

Reply:
Sort already exists when `NewCoins` is called